frontend/app/views/main.scala.html

```diff
@@ -25,6 +25,21 @@
         }
         <meta property="og:url" content="@(Config.membershipUrl + pageInfo.url)"/>
 
+        <script type="application/ld+json">
+            {
+                "@@context": "http://schema.org",
+                "@@type": "Organization",
+                "name": "Guardian Membership",
+                "url": "@(Config.membershipUrl)",
+                "logo": "@(Config.membershipUrl)@Asset.at("images/favicons/152x152.png")",
+                "sameAs" : [
+                    "http://www.twitter.com/@(Config.twitterUsername)",
+                    "@(Config.youtubeUrl)",
+                    "@(Config.googleplusUrl)"
```

Comment:
Actually, for templates I think "@(Config.googleplusUrl)" is actually better (it's shorter, less noise) - though the `s` prefix for string interpolation is a nice feature for Scala code files.

http://docs.scala-lang.org/overviews/core/string-interpolation.html